frontend/app/src/people/utils/__tests__/filterValidation.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { bountyHeaderFilter, bountyHeaderLanguageFilter } from '../filterValidation';
+
+describe('testing helpers', () => {
+  describe('bountyHeaderFilter', () => {
+    test('o/t/t', () => {
+      expect(bountyHeaderFilter({ Open: true }, true, true)).toEqual(false);
+    });
+    test('a/t/t', () => {
+      expect(bountyHeaderFilter({ Assigned: true }, true, true)).toEqual(true);
+    });
```

Comment:
looks like this should be `false`

Reply:
I wasn't sure because even if it's been paid, it's still assigned.